tests/thin-client/steel-thread.py

```diff
@@ -1,7 +1,48 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import logging
+
 from snowflake.snowpark import Session
-from snowflake.snowpark.functions import col, contains
 
-session = Session.builder.create()
-df = session.table('test_table')
-df = df.filter("STR LIKE '%e%'")
+logger = logging.getLogger(__name__)
+logging.basicConfig(
+    level=logging.DEBUG,
+    format="%(asctime)s [%(levelname)s] %(message)s",
+    handlers=[logging.StreamHandler()],
+)
+
+
+logger.info("Creating session")
+
+# Local devvm connection parameters.
+parameters = {
```

Comment:
I thought we didn't want these params in tests. E.g. this won't work with a cloud VM.

Reply:
sure, let me change it to the way it's currently handled for snowpark. Define a parameters.py file from which the variable is imported which isn't checked in.